CHANGES.md

```diff
@@ -1,4 +1,6 @@
 # UNRELEASED
+- [BREAKING CHANGE] Publish releases to https://github.com/IBM/cloudant-kafka-connector/releases.
+- [BREAKING CHANGE] Rename package from `com.ibm.cloudant.kafka` to `com.ibm.cloud.cloudant.kafka`.
```

Comment:
I wonder if we should specifically call this out and mention that users need to tweak their `connector.class` in any existing properties files.

Reply:
I can add more description here.
Given the list of breaking changes between what we have now and what is slated for `0.200` - I figure we'll need some kind of migration guide anyway.

Reply:
TBH I think you're right - we will have a tonne of stuff to write when the next release happens so I don't think it's a blocker right now.

Reply:
Added in 2a90c8f4858648f90043ad8a8b0d9f32e4f59aa8